.github/workflows/generic-amd64.yml

```diff
@@ -36,22 +36,35 @@ jobs:
     if: |
       (
         github.event.pull_request.head.repo.full_name == github.repository &&
-        github.event_name == 'pull_request'
+        github.event_name != 'pull_request_target'
```

Comment:
The comment says, 

`# prevent duplicate workflow executions for pull_request and pull_request_target`

So we are only preventing duplicate workflow executions for external contributors? That's what AFAIK are `pull_request_target` could be wrong.

Reply:
I added some detailed comments to explain 👍

Reply:
LGTM, see new comment in review below.